core/src/main/c/share/ooo.cpp

```diff
@@ -157,7 +157,7 @@ inline void radix_shuffle(uint64_t* counts, int64_t* src, int64_t* dest, uint64_
 #endif
 
 template<typename T>
-void radix_sort_long_index_asc_in_place(T *array, uint64_t size, T *cpy) {
+void radix_sort_long_index_asc_in_place(const T *array, uint64_t size, T *cpy) {
```

Comment:
`const T *array` used later as a `T *dest` for `radix_shuffle`:

`radix_shuffle<8u>(counts.c7, cpy, array, size);`